backend/server/sql/migrations/20250420_drv_build.sql

```diff
@@ -31,17 +31,6 @@ CREATE INDEX IF NOT EXISTS DrvBuildEventDerivation ON DrvBuildEvent (derivation)
 -- Speed up queries that want to retrieve all derivations in a specific state.
 CREATE INDEX IF NOT EXISTS DrvBuildEventState ON DrvBuildEvent (state);
 
--- For documentation, see the corresponding Rust struct.
-CREATE TABLE IF NOT EXISTS DrvRefs (
```

Comment:
Can you also remove the corresponding type from the build model? Or move it to the eval model?

Reply:
sure

Reply:
it's kind of it's own thing. how about drv model?

Reply:
Sounds good

Reply:
moved to drv.rs